src/main/java/org/spongepowered/api/stats/Statistic.java

```diff
@@ -40,13 +39,11 @@
     String getId();
 
     /**
-     * Gets the {@link StatisticFormat} of this statistic. If this is not
-     * present that this statistic's format is deferred to its group's default
-     * format.
+     * Gets the {@link StatisticFormat} of this statistic.
      *
-     * @return The format of this statistic, if available
+     * @return The format of this statistic
      */
-    Optional<StatisticFormat> getStatisticFormat();
+    StatisticFormat getStatisticFormat();
```

Comment:
The parent does not overwrite the child. It just states the format for all of its children. Without exceptions.

From my point the child should not overwrite what the parent says/states if it says/states anything.